streaming-pacman/pacman/game/js/pacman.js

```diff
@@ -267,10 +269,29 @@ function killingPacman() {
 		clearInterval(PACMAN_KILLING_TIMER);
 		PACMAN_KILLING_TIMER = -1;
 		erasePacman();
-		if (LIFES > 0) { 
+		if (LIFES > 0) { //TODO: need to add lifes = -1 event to user_game to define end of game
```

Comment:
What does this TODO means? Is it for a next iteration or you need to add something now? 
If is a future Idea maybe change from TODO to smg different